README.md

```diff
@@ -1847,14 +1847,14 @@ Other Style Guides
       return x + y;
     }
 
-    var x = 1;
-    var y = a + 2;
+    const x = 1;
```

Comment:
this isn’t removing *every* var in the document - either they’re all fine, or they all need to go, but we shouldn’t do it partially.